source/Pacioli/Pacioli.WebApi/Data/AccountantDbContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using Pacioli.WebApi.Models;
+
+namespace Pacioli.WebApi.Data
+{
+    public class AccountantDbContext : IdentityDbContext<Accountant>
```

Comment:
~~I don't think this is covered by tests, we don't need it yet.~~ 

Edit: nvm, I see that it is a required store for authentication.